grakn-test/test-integration/src/test/java/ai/grakn/test/migration/json/JsonMigratorMainTest.java

```diff
@@ -110,6 +109,7 @@ public void jsonMigratorCalledInvalidTemplateFile_ErrorIsPrintedToSystemErr(){
     }
 
     @Test
+    @Ignore
```

Comment:
TP task to undo this ignore?

Reply:
Well spotted, let me see if I can put it back